_assets/stylesheets/components/header.scss

```diff
@@ -39,3 +39,7 @@
     border: none;
   }
 }
+
+.usa-header {
+  margin-bottom: 4.8rem;
+}
```

Comment:
@jcbpl yeah, makes sense. Will update with my WIP on this branch. We'll hold off on the PR until I have the homepage stuff added.